bl/evaluation/evaluator.go

```diff
@@ -299,3 +377,19 @@ func countOccurrences(validationResult *Result) int {
 	}
 	return count
 }
+
+func extractSkipAnnotations(configuration extractor.Configuration) map[string]string {
+	skipAnnotations := make(map[string]string)
+	if configurationMetadata, ok := configuration["metadata"].(map[string]interface{}); ok {
+		if annotationsMap, ok := configurationMetadata["annotations"].(map[string]interface{}); ok {
+			for annotationKey, annotationValue := range annotationsMap {
+				if strings.Contains(annotationKey, SKIP_RULE_PREFIX) {
+					skipAnnotations[annotationKey] = annotationValue.(string)
+				}
+			}
+			return skipAnnotations
```

Comment:
I think this should be returned in the end of the function, even for readability purpose only